survey/survey-service-offering_shacl.ttl

```diff
@@ -0,0 +1,42 @@
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix survey: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/survey> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix general: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/general/> .
+@prefix gx: <https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#> .
+
+survey:SurveyServiceOfferingShape
+	a sh:NodeShape ; 
+	sh:targetClass survey:SurveyServiceOffering ; 
+	sh:property [
+		sh:path survey:surveyUrl ; 
+		sh:datatype xsd:string ; 
+		skos:example "'Example URL'" ; 
+		sh:minCount 1 ;
```

Comment:
i guess you should also add a maxCount = 1 here and at other properties. If you leave it blank you can specify more than one value for the property.